License2Deploy/rolling_deploy.py

```diff
@@ -250,6 +252,30 @@ def revert_deployment(self): #pragma: no cover
     logging.error("REVERT COMPLETE!")
     exit(self.exit_error_code)
 
+  def disable_all_cloudwatch_Alarms(self):
+    ''' Disable all the cloud watch alarms '''
+    cloud_watch_alarms = filter(lambda alarm: self.project in alarm.name and self.env in alarm.name, self.conn_cloudwatch.describe_alarms()) 
+    for alarm in cloud_watch_alarms:
```

Comment:
What do you think about this:

IF cloud_watch alarms exist, run through and disable them; otherwise, maybe just log that no cloud watch alarms exist and move on. I think these methods should be ran in the deploy method and therefor shouldn't force you to have to have cloud_watch_alarms enabled. Does that make sense?

Then, once you determine they exist, run the disable command for each alarm in a try block.